src/openrct2/core/String.hpp

```diff
@@ -184,4 +184,8 @@ namespace OpenRCT2::String
 
     // Escapes special characters in a string to the percentage equivalent that can be used in URLs.
     std::string URLEncode(std::string_view value);
+
+    int32_t StrLogicalCmp(char const* a, char const* b);
+    char* SafeStrCpy(char* destination, const char* source, size_t num);
+    char* SafeStrCat(char* destination, const char* source, size_t size);
```

Comment:
I think all of them should be renamed to something better now that its under the String namespace, also camelCase.

Reply:
I don't like the idea of mixing code styles, so for consistency I think I'll rename all of the functions in the String namespace to use lowerCamelCase in a separate commit then.